src/test/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataServiceTest.java

```diff
@@ -253,7 +253,8 @@ public void linkSealedClaimDocumentShouldReturnCaseDetails() {
 
         CaseDetails caseDetails = service.saveClaimDocuments(AUTHORISATION,
             SampleClaim.CLAIM_ID,
-            claim.getClaimDocumentCollection().orElse(new ClaimDocumentCollection()));
+            claim.getClaimDocumentCollection().orElse(new ClaimDocumentCollection()),
```

Comment:
Suggest to use orElseGet(ClaimDocumentCollection::new) to avoid unnecessary object instantiation.

Reply:
Agreed that although the comment is fair and good practice, it okay to leave it.